system_metrics_collector/test/system_metrics_collector/test_linux_process_cpu_measurement_node.cpp

```diff
@@ -60,6 +65,12 @@ class MockLinuxProcessCpuMeasurementNode : public system_metrics_collector::
    */
   double PeriodicMeasurement() override
   {
+    // first measurement does not add to the sample count
+    // proc cpu data requires two samples to compute a single measurement
+    if (++count_ > 1) {
```

Comment:
This will be true if count is >= 0. Is this intentional?

Reply:
sorry I mistread the code;
`++count > 1` does:
* preinc and return
* compare gt 1

so the first iteration its checking `1 > 1` which is `false`
and on the second iteration its checking `2 > 1` which is `true`